src/Render/Program.cs

```diff
@@ -104,8 +105,13 @@ internal static class Program {
 			string dstpath = Path.Join( context.OutputDirectory, relparent, $"{name}.{ext}" );
 			var outfile = new FileInfo( dstpath );
 
-			return new RelativeFile( context, infile, outfile );
-		}
+			string relativePath = Path.GetRelativePath( context.InputDirectory, path );
+			Uri absoluteEditUri = new Uri(
+				$"https://github.com/Brightspace/{context.DocRootRepoName}/edit/master/{relativePath}",

```

Comment:
Have any of our repos switched over to start using `main` as the main branch name? Maybe we could pull `GITHUB_BASE_REF` from the environment and put that on the `DocumentContext` as well?

Reply:
Oof, yes we have repos that use `main` instead of `master`. That's a good call.

Reply:
[Docs say that GITHUB_BASE_REF is only set on pull requests](https://docs.github.com/en/actions/reference/environment-variables), which won't do for our use here. Will have to think on this.

Reply:
#61 for later.